_i18n/template/resources/developer-guides/wallet-rpc.md

```diff
@@ -271,20 +329,25 @@ Send all unlocked balance to an address.
 Inputs:
 
 * *address* - string; Destination public address.
+* *account_index* - unsigned int; Sweep transactions from this account.
+* *subaddr_indices* - array of unsigned int; (Optional) Sweep from this set of subaddresses.
 * *priority* - unsigned int; (Optional)
-* *mixin* - unsigned int; Number of outpouts from the blockchain to mix with (0 means no mixing).
+* *mixin* - unsigned int; Number of outputs from the blockchain to mix with (0 means no mixing).
 * *unlock_time* - unsigned int; Number of blocks before the monero can be spent (0 to not add a lock).
 * *payment_id* - string; (Optional) Random 32-byte/64-character hex string to identify a transaction.
 * *get_tx_keys* - boolean; (Optional) Return the transaction keys after sending.
 * *below_amount* - unsigned int; (Optional)
 * *do_not_relay* - boolean; (Optional)
 * *get_tx_hex* - boolean; (Optional) return the transactions as hex encoded string.
+* *get_tx_metadata* - boolean; (Optional) return the transaction metadata as a string.
 
 Outputs:
 
 * *tx_hash_list* - array of string;
 * *tx_key_list* - array of string;
 * *tx_blob_list* - array of string; 
+* *tx_metadata_list* - array of string;
+* *tx_multisig_set* - string;
```

Comment:
Missing description (plus plenty below)

Reply:
This was to keep in line with the surrounding style (and got a bit lazy), I can add the description. Should I also add the other missing descriptions while I'm there?